plugins/chat/app/models/chat/message.rb

```diff
@@ -156,19 +156,8 @@ def cook
 
     def rebake!(invalidate_oneboxes: false, priority: nil)
       ensure_last_editor_id
-
-      previous_cooked = self.cooked
-      new_cooked =
-        self.class.cook(
-          message,
-          invalidate_oneboxes: invalidate_oneboxes,
-          user_id: self.last_editor_id,
-        )
-      update_columns(cooked: new_cooked, cooked_version: BAKED_VERSION)
-      args = { chat_message_id: self.id }
+      args = { chat_message_id: self.id, invalidate_oneboxes: invalidate_oneboxes }
       args[:queue] = priority.to_s if priority && priority != :normal
-      args[:is_dirty] = true if previous_cooked != new_cooked
-
```

Comment:
This was feeling un-necessary and duplicated, better let the job handle all this logic now.

Reply:
Yes that's the goal, it's not longer the role of rebake! to check for this, there's already a similar logic in the job to check of previous_cooked != new_cooked